test/unit/specs/utils/frequency.spec.js

```diff
@@ -0,0 +1,93 @@
+import { getSingleFrequencies, getDigramFrequencies, getTrigramFrequencies } from '@/utils/frequency'
```

Comment:
These aren't the worlds most perfect test cases... If you think they need to be better let me know : )